scripts/dotnet-cli-build/MsiTargets.cs

```diff
@@ -60,6 +69,20 @@ private static void AcquireWix(BuildTargetContext c)
             ZipFile.ExtractToDirectory(Path.Combine(WixRoot, "WixTools.zip"), WixRoot);
         }
 
+        private static void AcquireCrt(BuildTargetContext c)
+        {
+            if (File.Exists(CrtRedistributablePath))
+            {
+                return;
+            }
+
+            Directory.CreateDirectory(Path.GetDirectoryName(CrtRedistributablePath));
+
+            c.Info($"Downloading VC_redist.{Arch}.{CrtVersion}.exe..");
+
+            DownloadFile($"https://dotnetcli.blob.core.windows.net/build/vc/{Arch}/VC_redist.{Arch}.{CrtVersion}.exe", CrtRedistributablePath);
```

Comment:
> I do agree that we should get VCRedist from the MSFt location and not our CLI Blob - otherwise, we will be responsible for updating it and could endup installing a version that is older than what is available on VC Website.

Is there a well known URL for "give me the latest version of the Dev 14 CRT?"  The FW Links I've found are specific to a version of the CRT.  Not re-hosting it also means we don't ensure the dependency is transmitted over HTTPS end to end, which is a little frustrating.  It also makes us dependent on more external resources, which we don't want.

> How is the CLI acquired when you dont have Internet connection?

Having an internet connection when I download the CLI is different from a requirement to have an internet connection when I go to install the product.  Maybe I downloaded the CLI before getting on an airplane and then go to install it after we've taken off.